app-mvvm/AvaloniaAppTemplate.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
The .net core templates shouldn't be targeting net461 i don't think?

Reply:
my thinking on this was that if you then open in VS it wont work with previewer? or maybe that's no longer an issue?